src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -65,7 +67,17 @@ private static LinkedFile convert(List<String> entry) {
         while (entry.size() < 3) {
             entry.add("");
         }
-        LinkedFile field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
+        LinkedFile field;
+        if (LinkedFile.isOnlineLink(entry.get(1))) {
+            try {
+                field = new LinkedFile(new URL(entry.get(1)), entry.get(2));
+            } catch (MalformedURLException e) {
+                field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
+            }
+        } else {
+            field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
```

Comment:
Please don't use a try catch block as a normal control structure in the code.
https://github.com/HugoMatilla/Effective-JAVA-Summary#57-use-exceptions-only-for-exceptional-conditions

Reply:
I removed the assignment from the catch block and added an if-statement, which checks if *field* is *null* and sets a *LinkedFile* with a local file path. I hope that's what you meant.

Reply:
Yes, thanks for the addition. Lgtm